### ISSUE TITLE
JS error when calendar being destroyed in Angular, with custom content-injection

### ISSUE THREAD

**Issue by arshaw** [fullcalendar/fullcalendar#7105]
### Reduced Test Case

https://stackblitz.com/edit/github-msefaz?file=src%2Fapp%2Fmy-calendar%2Fmy-calendar.component.html

### Do you understand that if a reduced test case is not provided, we will intentionally delay triaging of your ticket?

- [X] I understand

### Which connector are you using (React/Angular/etc)?

Angular

### Bug Description

Reported by @hoangdinhcong in https://github.com/fullcalendar/fullcalendar/issues/7058#issuecomment-1360807386

With Angular, when using `eventContent`, and when the calendar is torn down, a JS error occurs.

**Comment by arshaw** [comment]
Fixed in [v6.0.2](https://github.com/fullcalendar/fullcalendar/releases/tag/v6.0.2)

Updated repro without errors:
https://stackblitz.com/edit/github-msefaz-k9zngw?file=src%2Fapp%2Fapp.component.ts,package.json

**Comment by joelcoxokc** [comment]
@arshaw 

I was able to fix the issue, by adding a boolean called `init` and wrapping the calendar in an ngIf. 
I then toggle the boolean in my ngOnInit()

```html
    <ng-container *ngIf="init">
        <full-calendar
            #calendar
            [options]="calendarOptions"
            [events]="data$ | async"></full-calendar>
    </ng-container>
```

```ts
public ngOnInit(): void {
    this.init = true;
}
```

Here is my calendar options

```ts
protected calendarOptions: CalendarOptions = {
    initialView: 'dayGridMonth',
    headerToolbar: false,
    locale: 'en',
    plugins: [dayGridPlugin, interactionPlugin],
    eventClick: (info) => this.handleEventClick(info),
    datesSet: (arg) => this.handleDateSet(arg),
};
```
